### PR TITLE
docs: add update conf

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: docs/zh/maintain/update_conf.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    ```bash
+    python tools/openmldb_ops.py --openmldb_bin_path=./bin/openmldb --zk_cluster=172.24.4.40:30481 --zk_root_path=/openmldb --cmd=showopstatus --filter=kFailed
+    ```
+* `showtablestatus`查看所有表状态是否正常
+    ```bash
+    python tools/openmldb_ops.py --openmldb_bin_path=./bin/openmldb --zk_cluster=172.24.4.40:30481 --zk_root_path=/openmldb --cmd=showtablestatus
+    ```
+一个tablet节点更新完成后，对其他tablet重复上述步骤。
+
+所有节点更新完成后恢复写操作, 执行`showtablestatus`命令查看`Rows`是否增加。
```
Comment (lumianph) [inline docs/zh/maintain/update_conf.md]: what is `Rows`?

Comment (dl239) [reply docs/zh/maintain/update_conf.md]: `Rows` is a column in the result of `showtablestatus`, which recored the total number records

Comment (lumianph) [reply docs/zh/maintain/update_conf.md]: 所有节点更新完成后恢复写操作，可以通过执行`showtablestatus`命令查看列 `Rows`，确认是否有新数据成功写入。

File: docs/zh/maintain/update_conf.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    ```
+    cp conf/tablet.flags conf/tablet.flags.bak
+    ```
+* 修改配置文件
+* 重启前准备`pre-upgrade`：为了避免对线上服务的影响，需要在重启tablet前，进行`pre-upgrade`操作，把该tablet上的leader分片迁移到其它tablets上（详细命令说明可以参考：[OpenMLDB运维工具](./openmldb_ops.md)）
+    ```bash
+    python tools/openmldb_ops.py --openmldb_bin_path=./bin/openmldb --zk_cluster=172.24.4.40:30481 --zk_root_path=/openmldb --cmd=pre-upgrade --endpoints=127.0.0.1:10921
+    ```
+  如果允许单副本表在升级更新中不可用，可以添加`--allow_single_replica`来避免添加新的副本。
+* 停止tablet
```
Comment (zhanghaohit) [inline docs/zh/maintain/update_conf.md]: 重启tablet

**Review by zhanghaohit — CHANGES_REQUESTED** [review]

**Comment by zhanghaohit** [comment]
shall we also add the doc on updating all configurations using `deploy-all.sh` scripts if it allows service unavailable during the updating?

- update the `conf/tablet.flags.template`, `conf/nameserver.flags.template`, `conf/apiserver.flags.template`, `conf/taskmanager.flags.template`
- `sbin/deploy-all.sh`
- `sbin/stop-all.sh` && `sbin/start-all.sh`

**Comment by lumianph** [comment]
> shall we also add the doc on updating all configurations using `deploy-all.sh` scripts if it allows service unavailable during the updating?
> 
> * update the `conf/tablet.flags.template`, `conf/nameserver.flags.template`, `conf/apiserver.flags.template`, `conf/taskmanager.flags.template`
> * `sbin/deploy-all.sh`
> * `sbin/stop-all.sh` && `sbin/start-all.sh`

yes, i think we can re-organize this doc to two sections, 1. configuration update without downtime (using the current content); 2. configuration update with restart

**Comment by dl239** [comment]
> shall we also add the doc on updating all configurations using `deploy-all.sh` scripts if it allows service unavailable during the updating?
> 
> * update the `conf/tablet.flags.template`, `conf/nameserver.flags.template`, `conf/apiserver.flags.template`, `conf/taskmanager.flags.template`
> * `sbin/deploy-all.sh`
> * `sbin/stop-all.sh` && `sbin/start-all.sh`

what's  the meaning of `allows service unavailable` ? The data in the storage need not to recovery ?

**Comment by zhanghaohit** [comment]
> > shall we also add the doc on updating all configurations using `deploy-all.sh` scripts if it allows service unavailable during the updating?
> > 
> > * update the `conf/tablet.flags.template`, `conf/nameserver.flags.template`, `conf/apiserver.flags.template`, `conf/taskmanager.flags.template`
> > * `sbin/deploy-all.sh`
> > * `sbin/stop-all.sh` && `sbin/start-all.sh`
> 
> what's the meaning of `allows service unavailable` ? The data in the storage need not to recovery ?

no. i mean the service is unavailable during the updating. That is, not serve any request during the update.

**Comment by dl239** [comment]
> > > shall we also add the doc on updating all configurations using `deploy-all.sh` scripts if it allows service unavailable during the updating?
> > > 
> > > * update the `conf/tablet.flags.template`, `conf/nameserver.flags.template`, `conf/apiserver.flags.template`, `conf/taskmanager.flags.template`
> > > * `sbin/deploy-all.sh`
> > > * `sbin/stop-all.sh` && `sbin/start-all.sh`
> > 
> > 
> > what's the meaning of `allows service unavailable` ? The data in the storage need not to recovery ?
> 
> no. i mean the service is unavailable during the updating. That is, not serve any request during the update.

- if users want to update the config of nameserver or apiserver only, it's not necessary  to restart the tablet
- `recoverdata` may not recover the data in some corner case. it's better to disable the `auto_failover` before stop all tablets.

**Review by zhanghaohit — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**